src/status_im2/contexts/chat/messages/content/album/view.cljs

```diff
@@ -13,39 +15,86 @@
   (when (= index 0) 12))
 
 (defn border-trr
-  [index]
-  (when (= index 1) 12))
+  [index count album-style]
+  (when (or (and (= index 1) (not= count 3))
+            (and (= index 0) (= count 3) (= album-style :landscape))
```

Comment:
The magical number 3 should preferably be extract to a var. It's used in lots of places, and if this number ever changes we want to be sure all usages change together.